its/plugin/plugins/php-custom-rules-plugin/src/main/java/org/sonar/samples/php/CustomPHPRulesDefinition.java

```diff
@@ -32,13 +32,13 @@ public String repositoryName() {
 
   @Override
   public String repositoryKey() {
-    return "php-custom-rules";
+    return "deprecated-php-custom-rules";
```

Comment:
@pynicolas how about renaming class as well?

Reply:
@pynicolas And it would be also nice to add comment explaining that it's deprecated

Reply:
Changed in 832a63fd.